components/shared/Page.js

```diff
@@ -16,18 +16,16 @@ function Title(props) {
   );
 }
 
-export default function Page({ children, fluid, hideSeparator, nav, title }) {
+export default function Page({ children, fluid, nav, title, isHome }) {
   let location = useRouter();
   const [currentPageRoute] = useState(location.pathname);
```

Comment:
this is a minor thing that was my mistake—we do not need `useState` here. you can just set the `NavComponent` prop to `location.pathname`.